SonarRunner.Shim/SonarRunner.Wrapper.cs

```diff
@@ -17,7 +18,13 @@ namespace SonarRunner.Shim
 {
     public class SonarRunnerWrapper : ISonarRunner
     {
-        private const int SonarRunnerTimeoutInMs = System.Threading.Timeout.Infinite;
+        private const string SonarOptsVariable = "SONAR_RUNNER_OPTS";
+
+        /// <summary>
+        /// Default value for the SONAR_RUNNER_OPTS
+        /// </summary>
+        /// <remarks>Reserving more than is available on the agent  </remarks>
```

Comment:
Reserving more than is available on the agent will cause the sonar-runner to fail